tests/flow/test_fulltext_query.py

```diff
@@ -57,4 +71,26 @@ def test01_fulltext_query(self):
 
         # fulltext query L2 for world 
         result = graph.query("CALL db.idx.fulltext.queryNodes('L2', 'world')")
-        self.env.assertEquals(result.result_set, [])
\ No newline at end of file
+        self.env.assertEquals(result.result_set, [])
+
+        # fulltext query L3 for redis and compare score
+        result = graph.query("CALL db.idx.fulltext.queryNodes('L3', 'redis') YIELD node, score RETURN node, score ORDER BY score DESC")
+        self.env.assertEquals(result.result_set[0][0].properties["v2"], "hello redis")
+        self.env.assertEquals(result.result_set[1][0].properties["v1"], "hello redis")
+        self.env.assertGreater(result.result_set[0][1], result.result_set[1][1])
+
+        expected_result = graph.query("MATCH (n:L4) RETURN n")
+
+        # fulltext query L4 for phelix
+        result = graph.query("CALL db.idx.fulltext.queryNodes('L4', 'phelix')")
```

Comment:
`felix` and `phelix` this is really cool!
Although the way the test construct its expectation doesn't implies the returned node actually contains the value `felix` I think a better construction of `expected_result` would be:
`expected_result = graph.query("MATCH (n {v:'felix'}) RETURN n)`

Reply:
L4 have only this node